src/mongo/MongoTripodConstants.php

```diff
@@ -76,3 +77,34 @@
 
 // Mongo constants
 define('DEFAULT_MONGO_CONNECT_TIMEOUT_MS', 20000);
+
+// queue name
+$tDiscoverQueueEnv = getenv("TRIPOD_DISCOVER_QUEUE");
```

Comment:
Add these env vars to the build file so it fails to do an ```ant init``` (or grunt when we move to it) if they aren't set?